autogpt_platform/backend/backend/monitoring/notification_monitor.py

```
@@ -1,38 +1,28 @@
+    """Send everything that has sat out the ten-minute debounce window."""
     try:
-        logging.info(
-            f"Processing existing batches for notification type {args.notification_types}"
-        )
-        get_notification_manager_client().process_existing_batches(
-            args.notification_types
-        )
+        get_notification_manager_client().flush_matured_alerts()
     except Exception as e:
-        logger.exception(f"Error processing existing batches: {e}")
+        logger.exception(f"Error flushing matured alerts: {e}")
```

Ruff (0.16.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)